libfuse/mounter_osx.go

```diff
@@ -51,6 +54,7 @@ func GetPlatformSpecificMountOptionsForTest() []fuse.MountOption {
 	return []fuse.MountOption{
 		fuse.OSXFUSELocations(kbfusePath, fuse.OSXFUSELocationV3),
 		fuse.ExclCreate(),
+		// fuse.LocalVolume(),
```

Comment:
Tests are passing; they just leave behind many items under Devices in the Finder sidebar. But I'll try to fix that